### ISSUE TITLE
Bug: excludedTypes config setting doesn't seem to work

### ISSUE THREAD

**Issue by Krislunde** [boazpoolman/strapi-plugin-sitemap#150]
### Describe the bug

When configuring the plugin in plugins.js, the `excludedTypes` key doesn't actually seem to work. The documentation doesn't actually make it clear what the format should be here either. Is it the name that shows up in the url bundle list in the strapi admin, or is it the api id for the content type, or something else? In my case, I can't get any of my content types to disappear from the list of url bundles.

### Steps to reproduce the behavior

1. Create a content type
2. Try to exclude it through the `excludedTypes` config key.
3. Observe no types being removed from list, even after rebuilding strapi

### Expected behavior

The content type should disappear from list as an option

**Comment by MSACC** [comment]
Following this file 

https://github.com/boazpoolman/strapi-plugin-sitemap/blob/60e22997c4dde62744f0fd132191b6acc6eec708/server/config.js#L11

The excludedTypes in plugins.js / plugin.ts is meant to exclude modules from the plugin.

After adding this to my plugins.ts file:
<img width="525" alt="image" src="https://github.com/boazpoolman/strapi-plugin-sitemap/assets/35879908/2fa85564-7ec9-472a-9337-36fd8c1c1801">

I wasn't able anymore to choose the content type in my url bundles:
<img width="1046" alt="image" src="https://github.com/boazpoolman/strapi-plugin-sitemap/assets/35879908/8331a423-2594-4fc7-8907-62639175a8e2">

It won't delete the content type from the sitemap. You first need to delete/change the URL bundle before this will affect your sitemap.

TLDR: excludedTypes is something you use for excluding in plugin to use from, but when updated after used already, it will still be used in the sitemap.

**Comment by Krislunde** [comment]
> excludedTypes is something you use for excluding in plugin to use from

Yes, I wasn't trying to remove it fromt he sitemap, I was trying to remove the content type from the url bundles the plugin allows you to choose from. Thank you. The documentation didn't explain the format required. I tried adding:

1) ['content-type']
2) ['content-type']
3) ['contentType.contentType']

and probably a few others. I guess it was the api:: part that was needed. I'll give it a try and report back later.

This should probably be shown in the documentation as an example.

**Comment by boazpoolman** [comment]
The documentation about this got updated in [this PR](https://github.com/boazpoolman/strapi-plugin-sitemap/pull/153).

**Comment by boazpoolman** [comment]
@Krislunde Can you confirm that (with the proper documentation) the setting does actually work on your end?

**Comment by boazpoolman** [comment]
This was released with version `3.0.7` of the plugin
